bin/node-pg-migrate.ts

```diff
@@ -21,10 +21,7 @@ process.on('uncaughtException', (err) => {
   process.exit(1);
 });
 
-const crossRequire = createRequire(
-  // @ts-expect-error: ignore until esm only
-  import.meta.url || __dirname
-);
+const crossRequire = createRequire(resolve('_'));
```

Comment:
@aprendendofelipe I've been following the with interest the work on ESM support here. I was reading the code but can't figure out what this does. Specifically, is this underscore (`_`) a special directory?

Can you help just so that I earn something today 😉🙏

Reply:
It is explained in the thread https://github.com/salsita/node-pg-migrate/pull/1206#discussion_r1646530197